src/ds/json/intro.md

```diff
@@ -0,0 +1,17 @@
+The JSON capability of Redis Stack provides JavaScript Object Notation (JSON) support for Redis, which allows Redis to function as a document database.
```

Comment:
It is worth noting that each JSON key stores a single JSON document, and together with Search and query it allows Redis to function as a document database.